ros2_control_demos/package.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0"?>
+<package format="3">
+  <name>ros2_control_demos</name>
+  <version>0.0.0</version>
+  <description>Metapackage for ROS2 control demos packages</description>
+
+  <author email="ostasse@laas.fr">Olivier Stasse</author>
+  <author email="denis@stogl.de">Denis Štogl</author>
+
+  <maintainer email="allabana@amazon.com">Anas Abou Allaban</maintainer>
```

Comment:
The guys from AWS won't take part in maintenance, please remove this line

Reply:
Done